rhtconsulting/rhc-ose
# Changed to allow for separate /var/log partition

#### What does this PR do?

The tools now allow for a separate /var/log partition on the additional volume attached to the instances. If no configuration is passed, /var/log will stay as-is - i.e.: co-existing with the root file system in the partition used for /. If "CONF_STORAGE_SIZE_LOGGING" is set greater than 0, a partition of X GB will be allocated for logs.
#### How should this be manually tested?

**NOTE:** Make sure to run the tests on an image generated by the latest image creation process.
1. Create a config file with CONF_STORAGE_SIZE_LOGGING=X, where X is the number of GB to allocate for a log partition.
2. Run "./provisioning/osc-provision --num-nodes=2 --config=<path_to_config_file> --key=<public_key>"
#### Is there a relevant Issue open for this?
#67
#### Who would you like to review this?

/cc @etsauer
